docs/en/enterprise-edition/policy-reference/aws-policies/aws-iam-policies/ensure-the-aws-execution-role-arn-and-task-role-arn-are-different-in-ecs-task-definitions.adoc

```diff
@@ -23,8 +23,32 @@
 |=== 
 
 
+=== Description 
 
-=== Description
+In Amazon ECS, the execution role and the task role serve different purposes and should be distinct:
```

Comment:
In Amazon ECS, it's important to distinguish between the execution task roles as they fulfill separate roles and serve distinct purposes: